packages/web3-providers-ws/src/helpers.js

```diff
@@ -19,9 +19,7 @@ if (isNode) {
     }
 } else {
     _btoa = btoa.bind(window);
-    helpers = function(url) {
-        return new URL(url);
-    };
+    helpers = require('url').parse;;
```

Comment:
Small typo here with the double semicolons

```suggestion
    helpers = require('url').parse;
```